hw5/src/GrowListTest.java

```diff
@@ -0,0 +1,39 @@
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+import java.util.Iterator;
+import java.util.NoSuchElementException;
+
+public class GrowListTest<E> {
+	GrowList<String> g = new GrowList<String>();
+
+	@Test(expected = UnsupportedOperationException.class)
+	public void testRemove() {
+		g.add("banana");
+		g.add("apple");
+		g.add("oranges");
+		
+		Iterator<String> iter = g.values();
+		iter.next();
+		iter.remove();
+	}
+
+	@Test public void testHasNext() {
+		g.add("x");
+		g.add("y");
+		g.add("z");
+		Iterator<String> iter = g.values();
+		iter.next();
+		assertEquals(true, iter.hasNext());
+		iter.next();
+		iter.next();
+		assertEquals(false, iter.hasNext());
+
+	}
+
+    @Test(expected = NoSuchElementException.class)
```

Comment:
Since `testHasNext()` tests `next()`, I figure this test would be for the exception. Anything else I am missing?